src/ci/docker/dist-various-2/Dockerfile

```diff
@@ -65,6 +70,10 @@ ENV TARGETS=$TARGETS,wasm32-unknown-unknown
 ENV TARGETS=$TARGETS,x86_64-sun-solaris
 ENV TARGETS=$TARGETS,x86_64-unknown-linux-gnux32
 ENV TARGETS=$TARGETS,x86_64-unknown-cloudabi
+ENV TARGETS=$TARGETS,x86_64-fortanix-unknown-sgx
+ENV TARGETS=x86_64-fortanix-unknown-sgx
```

Comment:
I think this is not correct

Reply:
Thanks for the review. I had added it for faster testing. Forgot to remove it.
Pushed the fix.